internal/security/proxy/token.go

```diff
@@ -0,0 +1,40 @@
+/*******************************************************************************
+ * Copyright 2019 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ * @author: Tingyu Zeng, Dell
+ *******************************************************************************/
+
+package proxy
+
+import (
+	"encoding/json"
+	"io"
+)
+
+type UserTokenPair struct {
+	User  string
+	Token string
+}
+
+func (ut *UserTokenPair) Save(w io.Writer) error {
+	jdata, err := json.MarshalIndent(*ut, "", " ")
+	if err != nil {
+		return err
+	}
+
+	_, err = w.Write(jdata)
```

Comment:
minor nitpick this could just be 
```suggestion
	_, err = w.Write(jdata)
	return err
```

and just don't check err at all